rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/openstack/OpenstackBakeHandlerSpec.groovy

```diff
@@ -116,564 +115,582 @@ class OpenstackBakeHandlerSpec extends Specification {
+        ==> openstack: Deleting temporary keypair: packer 573d433f-278f-3e22-d2ce-e5720f906cf8 ...
+        Build 'openstack' finished.
+
+        ==> Builds finished. The artifacts of successful builds are:
+        --> openstack: An image was created: 1f28b46b-b36f-4b7c-bc34-40e2371886fa
+      """
+
+      Bake bake = openstackBakeHandler.scrapeCompletedBakeResults(REGION, "123", logsContent)
 
     then:
-    with(bake) {
-      id == "123"
-      !ami
-      image_name == "1f28b46b-b36f-4b7c-bc34-40e2371886fa"
-    }
+      with(bake) {
+        id == "123"
+        !ami
+        image_name == "1f28b46b-b36f-4b7c-bc34-40e2371886fa"
+      }
   }
 
   void 'scraping returns null for missing image name'() {
     setup:
-    @Subject
-    OpenstackBakeHandler openstackBakeHandler = new OpenstackBakeHandler(openstackBakeryDefaults: openstackBakeryDefaults)
+      @Subject
+      OpenstackBakeHandler openstackBakeHandler = new OpenstackBakeHandler(openstackBakeryDefaults: openstackBakeryDefaults)
 
     when:
-    def logsContent = """
-==> openstack: Discovering enabled extensions...
-==> openstack: Loading flavor: smem-2vcpu
-    openstack: Verified flavor. ID: 89a74515-7072-49f8-87ab-28d3f70de59d
-==> openstack: Creating temporary keypair: packer 573d433f-278f-3e22-d2ce-e5720f906cf8 ...
-==> openstack: Created temporary keypair: packer 573d433f-278f-3e22-d2ce-e5720f906cf8
-==> openstack: Launching server...
-    openstack: Server ID: 732520f1-af79-45be-bcda-ea1089e80f0d
-==> openstack: Waiting for server to become ready...
-==> openstack: Creating floating IP...
-    openstack: Pool: ext_vlan1472_net
-    openstack: Created floating IP: 1.2.3.4
-==> openstack: Associating floating IP with server...
-    openstack: IP: 1.2.3.4
-    openstack: Added floating IP 1.2.3.4 to instance!
-==> openstack: Waiting for SSH to become available...
-==> openstack: Connected to SSH!
-==> openstack: Pausing 30s before the next provisioner...
-==> openstack: Provisioning with shell script: rosco-web/config/packer/install_packages.sh
-    openstack: repository=
-    openstack: package_type=
-    openstack: packages=
-    openstack: upgrade=
-==> openstack: Stopping server: 732520f1-af79-45be-bcda-ea1089e80f0d ...
-    openstack: Waiting for server to stop: 732520f1-af79-45be-bcda-ea1089e80f0d ...
-==> openstack: Creating the image: dan-packer-test-3
-    openstack: Image: 1f28b46b-b36f-4b7c-bc34-40e2371886fa
-==> openstack: Waiting for image dan-packer-test-3 (image id: 1f28b46b-b36f-4b7c-bc34-40e2371886fa) to become ready...
-==> openstack: Terminating the source server: 732520f1-af79-45be-bcda-ea1089e80f0d ...
-==> openstack: Deleting temporary keypair: packer 573d433f-278f-3e22-d2ce-e5720f906cf8 ...
-Build 'openstack' finished.
-
-==> Builds finished. The artifacts of successful builds are:
-"""
-
-    Bake bake = openstackBakeHandler.scrapeCompletedBakeResults(REGION, "123", logsContent)
+      def logsContent = """
```

Comment:
oh, yes those indents shouldn't have been made part of the actual string. I'll strip the margins so we can still keep the indentation.